seacrowd/sea_datasets/tydiqa/tydiqa.py

```diff
@@ -0,0 +1,422 @@
+                       subset_id="tydiqa_secondary_task_indonesian"),
+    ]
+    DEFAULT_CONFIG_NAME = "tydiqa_primary_task_source"
+
+    def _info(self):
+        if "primary_task" in self.config.name:
+            if "source" in self.config.name:
+                return datasets.DatasetInfo(
+                    description=_DESCRIPTION,
+                    features=datasets.Features(
+                        {
+                            "passage_answer_candidates": datasets.features.Sequence(
+                                {
+                                    "plaintext_start_byte": datasets.Value("int32"),
+                                    "plaintext_end_byte": datasets.Value("int32"),
+                                }
+                            ),
+                            "question_text": datasets.Value("string"),
+                            "document_title": datasets.Value("string"),
+                            "language": datasets.Value("string"),
+                            "annotations": datasets.features.Sequence(
+                                {
+                                    "passage_answer_candidate_index": datasets.Value("int32"),
+                                    "minimal_answers_start_byte": datasets.Value("int32"),
+                                    "minimal_answers_end_byte": datasets.Value("int32"),
+                                    "yes_no_answer": datasets.Value("string"),
+                                }
+                            ),
+                            "document_plaintext": datasets.Value("string"),
+                            "document_url": datasets.Value("string")
+                        }
+                    ),
+                    citation=_CITATION,
+                )
+            elif "seacrowd" in self.config.name:
+                features = schemas.qa_features
+                features["meta"] = {
+                    "passage_answer_candidates": datasets.features.Sequence(
+                        {
+                            "plaintext_start_byte": datasets.Value("int32"),
+                            "plaintext_end_byte": datasets.Value("int32"),
+                        }
+                    ),
+                    "annotations": datasets.features.Sequence(
+                        {
+                            "passage_answer_candidate_index": datasets.Value("int32"),
+                            "minimal_answers_start_byte": datasets.Value("int32"),
+                            "minimal_answers_end_byte": datasets.Value("int32"),
+                            "yes_no_answer": datasets.Value("string"),
+                        }
+                    ),
+                    "language": datasets.Value("string"),
+                }
+
+                return datasets.DatasetInfo(
+                    description=_DESCRIPTION,
+                    features=features,
+                    citation=_CITATION,
+                )
```

Comment:
Let's keep the `datasets.DatasetInfo` information as is in the template:

> return datasets.DatasetInfo(
>             description=_DESCRIPTION,
>             features=features,
>             homepage=_HOMEPAGE,
>             license=_LICENSE,
>             citation=_CITATION,
>         )

same goes for the one in `secondary task`